.changeset/sharp-planes-turn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-vault-backend': patch
```

Comment:
We should make this a `minor` as there are some potentially breaking changes with the removed types, I'm not sure if it's even likely that someone uses them but at least to indicate.

Reply:
That's right. Just changed it to `minor`